### PR TITLE
Accessiblity

### PR DESCRIPTION
**Pull request by eng-mohamed-eldeeb**
An accessibility report as we determined:

- Page titles - one issue found [link](https://github.com/eng-mohamed-eldeeb/protofolio/blob/4e73662cd1de249fdf23c0fa5c28999d00f44d21/index.html#L22)
-  Image text alternatives - four issues found:
    1. [link](https://github.com/eng-mohamed-eldeeb/protofolio/blob/e97c9e5292d47885c85158b870919c22e833615b/index.html#L89)
    2. [link](https://github.com/eng-mohamed-eldeeb/protofolio/blob/e97c9e5292d47885c85158b870919c22e833615b/index.html#L110)
   3. [link](https://github.com/eng-mohamed-eldeeb/protofolio/blob/e97c9e5292d47885c85158b870919c22e833615b/index.html#L132)
   4. [link](https://github.com/eng-mohamed-eldeeb/protofolio/blob/e97c9e5292d47885c85158b870919c22e833615b/index.html#L152)
- Text headings - no issues found
- Color contrast - one issue found [link](https://github.com/eng-mohamed-eldeeb/protofolio/blob/e97c9e5292d47885c85158b870919c22e833615b/style.css#L243-L244)
- Resize - no issue found
- Interaction - no issue found
- Moving content - no issue found
- Multimedia - no issue found
- The basic structure of the page -no issue found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
       integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
       crossorigin="anonymous"
     />
-    <title>Protfolio</title>
+    <title>Mohamed's Portfolio homepage</title>
```
Comment (elmar8287) [inline index.html:22]: - [ ] If you click on line 22 or 23, or any other line, you can see the URL changing. For example, by clicking line 22 we can see the URL https://github.com/eng-mohamed-eldeeb/protofolio/pull/5/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L22. You can copy this link for your report, for your the first point:

- Page titles - one issue found [link](https://github.com/eng-mohamed-eldeeb/protofolio/pull/5/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L22)

Please, consider this technic for all found issues.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @eng-mohamed-eldeeb, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  Check the comments under the review.
- [ ]  When an issue is found please link to the corresponding line of code in the Pull Request changes. So it's easy to trace the issue. You can familiar how to do it via [this link](http://www.chasewoodford.com/blog/linking-highlighted-lines-code-github/#:~:text=To%20link%20to%20multiple%20lines,php%23L117%2DL148)
- [ ] Please, consider the project requirements, which mentioned that you should prepare the report, listing all points that you have checked, and the result of your evaluation. So, it means your report will view something like the following:

1. Page titles - one issue found
2. Image text alternatives - one issue found (link to the corresponding line of code as I described upper)
3. Text headings - no issue found
4. Color contrast - no issue found
5. Resize - no issue found
6. Interaction - no issue found
7. Moving content - one issue found
8. Multimedia - no issue found 
9. The basic structure of the page - no issue found

(this is a template for you)

- [x] Please, consider, that this report should be in the description of PR. You described it in description 👍, but please consider the report form, I mentioned, as the project required.
- [x] Please, consider, that you need to use pair-programming. That is why you need to work together with your partner and it will also be shown in commits history.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (AckonSamuel) [reply index.html:22]: thank you

Comment (elmar8287) [reply index.html:22]: @eng-mohamed-eldeeb You are welcome.

Comment (AckonSamuel) [reply index.html:22]: Please, we have made all the changes you requested.

**Review by Whoistolu — APPROVED** [review]
### Hi team,

Thank you for implementing the required changes.
Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (elmar8287) [reply index.html:22]: @AckonSamuel Nice job done 👍

**Pull request merged**